ui/job-view/details/summary/ActionBar.jsx

```diff
@@ -423,7 +429,6 @@ class ActionBar extends React.PureComponent {
       currentRepo,
     } = this.props;
     const { customJobActionsShowing } = this.state;
-
```

Comment:
We should leave this blank line.  It's a general convention to have a blank line between ``const``s and functionality.  There are places in the code that haven't been fixed for this yet, but let's stick with it where we can.

Reply:
Fixed